tests/integration/suite/test_workloads.py

```diff
@@ -310,6 +310,31 @@ def test_statefulset_workload_volumemount_subpath(admin_pc):
         assert e.value.error.status == 422
 
 
+def test_workload_redeploy(admin_pc):
+    client = admin_pc.client
+    ns = admin_pc.cluster.client.create_namespace(
+        name=random_str(),
+        projectId=admin_pc.project.id)
+    name = random_str()
+    workload = client.create_workload(
+        name=name,
+        namespaceId=ns.id,
+        scale=1,
+        containers=[{
+            'name': 'one',
+            'image': 'nginx',
+        }],
+        annotations={
+            "cattle.io/timestamp":
+                datetime.datetime.now().isoformat()}, )
+    client.action(workload, "redeploy")
+    timestamp = client.list_workload(
```

Comment:
do we need some retry/sleep/check logic around this? might lead to error in drone

Reply:
I agree with @mrajashree this really needs retry as the annotation may not be set right away